src/Symfony/Component/Cache/CacheInterface.php

```diff
@@ -1,37 +0,0 @@
-<?php
-
-/*
- * This file is part of the Symfony package.
- *
- * (c) Fabien Potencier <fabien@symfony.com>
- *
- * For the full copyright and license information, please view the LICENSE
- * file that was distributed with this source code.
- */
-
-namespace Symfony\Component\Cache;
-
-/**
- * Gets and stores items from a cache.
- *
- * On cache misses, a callback is called that should return the missing value.
- * It is given two arguments:
- * - the missing cache key
- * - the corresponding PSR-6 CacheItemInterface object,
- *   allowing time-based expiration control.
- *
- * @author Nicolas Grekas <p@tchwork.com>
- */
-interface CacheInterface
```

Comment:
For the record: This interface has never been released, so it is okey to remove it.